kalite/i18n/management/commands/makemessages.py

```diff
@@ -11,16 +11,27 @@
 from django.core.management.commands import makemessages
 
 
+IGNORE_PATTERNS = [
+    "*/node_modules*",
+    "*dist-packages*",
+    "*python-packages*",
+    "*/kalite/static*",
+    "*bundle*.js",
+    "*/ka-lite/build/*",
+    "*/js/i18n/*.js",
+    "*/ka-lite/docs/*"
```

Comment:
I'm withholding the internationalization of docs for now. Takes a bit of thinking through before we should go ahead with it.